docs/system_requirements/index.sdoc

```diff
@@ -91,14 +91,36 @@ UID: ZEP-SYRS-7
 STATUS: Draft
 TYPE: High Level
 COMPONENT: Interrupts
-TITLE: Interrupt Service Routine
+TITLE: Interrupt Management
 STATEMENT: >>>
-The Zephyr RTOS shall provide a framework for interrupt management.
+The Zephyr RTOS shall provide a framework for interrupt and interrupt service routine management.
 <<<
 USER_STORY: >>>
-As a Zephyr RTOS user I want interrupts to be handled synchronously in response to a hardware or software interrupt request with a minimum latency, preempting threads and, as far as the hardware allows, lower priority interrupt service routines.
+As the Zephyr RTOS user I want the Kernel to provide abstracted interfaces to 
+the platform enabling me to implement standard interrupts interrupt service routines 
+without detailed knowledge of the platform architecture and programming model. 
 <<<
 
+[REQUIREMENT]
+UID: ZEP-39-1
```

Comment:
This Req UID don't align with the System requirements UIDs Layout

Reply:
Not sure how to handle this.  It is a system requirement and links to ZEP-SYRS-7 but no others.
Do I use the next available ZEP-SYRS-#, or do I use -1?

Either works for me, what would you prefer.
I used ZEP-SYRS-7-1 to make strictdoc run.

Reply:
The easiest way is to delete the line with the UID locally and than execute the strict doc command `strictdoc manage auto-uid .` Than the next free SYRS UID is selected or as you described it using the next available TEP-SYRS-#